generator/test/error_test_data/not_an_indexable_type.expected

```diff
@@ -1,4 +1,4 @@
-Realm only supports the @Indexed() annotation on fields of type
+Realm only supports the @Indexed annotation on fields of type
```

Comment:
You actually have to write the `()` in dart as you const constructing an object, but I guess it is okay for error message

Reply:
Yeah, I removed it from the error because now you can also do `@Indexed(RealmIndexType.fullText)` and figured it may be more confusing if the error message removed the argument than if it didn't include the parentheses at all.